Source/Core/Common/x64Emitter.cpp

```diff
@@ -309,7 +309,7 @@ void OpArg::WriteRest(XEmitter* emit, int extraBytes, X64Reg _operandReg,
     s64 distance = (s64)offset - (s64)ripAddr;
     ASSERT_MSG(DYNA_REC,
                (distance < 0x80000000LL && distance >= -0x80000000LL) || !warn_64bit_offset,
-               "WriteRest: op out of range (0x%" PRIx64 " uses 0x%" PRIx64 ")", ripAddr, offset);
```

Comment:
Do we still need `#include <cinttypes>` after removing this use of `PRIx64`?

Reply:
Nope.  Same applies to Arm64Emitter.

In fact, all inclusions of `<cinttypes>` are unused other than the ones in MenuBar (where it is used to create title IDs) in RenderBase (where it is used with ImGui, which uses printf strings).  I've added another commit to remove these.